certified-connectors/jbhunt/README.md

```diff
@@ -0,0 +1,26 @@
+This connector was specifically developed to be used for Dynamics 365 Intelligent Order Management's JBHunt provider integration.
```

Comment:
@vechall Please provide your connector description in more in-detailed which should describe concisely the main purpose and value offered by your connector.